### DISCUSSION TITLE
IOS version

### DISCUSSION THREAD

**Discussion by villyaraujo** [maxrave-dev/SimpMusic#360] [Ideas]
Are there plans for a version for IOS or Flutter?

**Comment by maxrave-dev** [comment]
Need money to buy MacBook 💻🤧🤧🤧

**Reply by Zdertis420** [reply]
As I know you can use virtual machine but I'm not sure....

**Reply by dl4194** [reply]
So can you build this for iOS if you have a Mac.

**Reply by Rafee888** [reply]
Need ios version please 🥺

**Reply by xwastyz** [reply]
What if you hackintosh some random laptop you might have laying around? 👀

**Comment by villyaraujo** [comment]
I understand, but can you recreate this App in Flutter? If yes, how much would it cost for the service?

**Reply by maxrave-dev** [reply]
Hmm, Flutter need Macbook to build iOS app. So anyway, I need to go to office and spend to buy a Macbook

**Reply by Rafee888** [reply]
Bro . Please do it

**Comment by Rafee888** [comment]
Need ios version

**Comment by Zigo10** [comment]
So I did some research (I Will allegate screenshot, yes, I used Gemini 3 pro for researching but I will dig deeper) and there's actually hope for an ios version especially in EU. Hope this helps!

![Screenshot_2026-02-15-11-52-26-177_com.discord.jpg](https://github.com/user-attachments/assets/87b05e1b-a88a-4b38-9cfa-028c37068962)

![Screenshot_2026-02-15-11-52-33-274_com.discord.jpg](https://github.com/user-attachments/assets/1020fef3-39c2-4179-b190-2f0622ed717e)

![Screenshot_2026-02-15-11-52-28-807_com.discord.jpg](https://github.com/user-attachments/assets/8e43e3a0-26ef-49de-90b6-1a952765a17e)

And I found out that there is an open source app for music in iOS , which is useful so can "copy" all the audio engine etc.  And  he has to "just" make (I know it's a lot of work) an  .IPA version and then we have to manually install it

![Screenshot_2026-02-15-11-54-58-863_com.discord.jpg](https://github.com/user-attachments/assets/ffa23ac3-29fc-43a4-a6e5-f826e9695284)

![Screenshot_2026-02-15-11-55-02-710_com.discord.jpg](https://github.com/user-attachments/assets/50dfde96-dc32-4fac-9a2b-366ca133dfcc)

**Comment by Zigo10** [comment]
Ps I found that in AppStore there's an app called demus, it open surce and uses YouTube in back end, @maxrave-dev  you can take some inspiration about the code for iOS, but I'll tell you something, you will still won the competition because your UI it's divine, you have liquid glass, and many setting(es quality setting between download and online playing), charts, Spotify canvas, and good playlist organization
